src/trunk.c

```diff
@@ -1186,18 +1210,27 @@ trunk_inc_pivot_generation(trunk_handle *spl, page_handle *node)
    return hdr->pivot_generation++;
 }
 
+/*
+ * A pivot consists of the pivot key (of size cfg.key_size) followed by
+ * a struct trunk_pivot_data. Return the total size of a pivot.
+ */
 uint64
 trunk_pivot_size(trunk_handle *spl)
 {
    return trunk_key_size(spl) + sizeof(trunk_pivot_data);
 }
 
+/* struct trunk_pivot_data{} itself is the message. Return its size */
```

Comment:
This naming is confusing out of context and the comment doesn't really help. Maybe @rtjohnso could suggest something.